deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -251,6 +260,9 @@ serviceMonitors:
             [__meta_kubernetes_pod_label_openebs_io_persistent_volume_claim]
           action: replace
           targetLabel: openebs_pvc
+        - sourceLabels: [__meta_kubernetes_pod_label_openebs_io_controller]
```

Comment:
is there any jiva specific label? 

For example, can we keep the `openebs.io/cas-type` label and use the value as jiva/cstor in the queries ?